Api/NodeRepositoryInterface.php

```diff
@@ -6,15 +6,42 @@
 
 interface NodeRepositoryInterface
 {
+    /**
+     * @param \Snowdog\Menu\Api\Data\NodeInterface $page
+     * @return \Snowdog\Menu\Api\Data\NodeInterface
+     */
     public function save(NodeInterface $page);
 
+    /**
+     * @param int $id
+     * @return \Snowdog\Menu\Model\Menu\Node
+     */
     public function getById($id);
 
+    /**
+     * @param \Magento\Framework\Api\SearchCriteriaInterface $criteria
+     * @return \Magento\Framework\Api\SearchResultsInterface
+     */
     public function getList(SearchCriteriaInterface $criteria);
 
+    /**
+     * @param \Snowdog\Menu\Api\Data\NodeInterface $page
+     * @return bool
+     */
     public function delete(NodeInterface $page);
 
+    /**
+     * @param int $id
+     * @return bool
+     */
     public function deleteById($id);
 
+    /**
+     * Return node by menu id
+     *
+     * @api
+     * @param int $menuId
+     * @return \Snowdog\Menu\Api\Data\NodeInterface[]
+     */
     public function getByMenu($menuId);
```

Comment:
Should't this method be renamed to `getByMenuId`?

Reply:
updated